src/SUMMARY.md

```diff
@@ -7,7 +7,8 @@
     - [Paměť](uvod/pamet.md)
 - [Nastavení prostředí](prostredi/nastaveni_prostredi.md)
     - [Linux](prostredi/linux/linux.md)
-      - [Instalace](prostredi/linux/instalace.md) 
+      - [Instalace](prostredi/linux/instalace.md)
```

Comment:
I think I would like to keep macOS out of the fast path. Let's modify it like this:

```
- [Operační systém]()
  - [Instalace Linuxu]()
  - [Nastavení macOS]()
```